docs/getting-started/which-permissions-does-codacy-need-from-my-account.md

```diff
@@ -63,15 +63,15 @@ If you log in with GitHub, Codacy requires the following [app permissions](https
       <td>Read & Write</td>
       <td>Codacy sets the status of commits according to the result of code analysis.</td>
     </tr>
-    <tr>
-      <td>Administration</td>
-      <td>Read & Write</td>
-      <td><a href="#why-does-codacy-ask-for-permission-to-create-ssh-keys">Codacy creates an SSH key</a> on the repository to allow cloning and integrating with your repository.</td>
-    </tr>
     <tr>
       <td>Contents</td>
       <td>Read-Only</td>
-      <td>Codacy accesses repository contents to provide faster code coverage analysis and as part of an initiative to use GitHub App tokens instead of SSH keys when cloning repositories for code quality analysis.</td>
+      <td>Codacy uses installation access tokens when integrating with your repositories and cloning them for code quality analysis, and retrieves repository contents to provide faster code coverage analysis.</td>
```

Comment:
@andrzej-janczak @lorandszakacs I did a tweak here to mention **installation access tokens**. Let me know if it's correct. 🙏 
I want to confirm if Codacy **uses** installation access tokens. (instead of creating/generating... for that maybe we would need more than read-only access, right?)

Reply:
look ok 👍